clients/drcachesim/tests/scheduler_unit_tests.cpp

```diff
@@ -406,6 +406,10 @@ test_regions_bare_no_marker()
         /* clang-format off */
         make_thread(1),
         make_pid(1),
+        /* Avoid reader_t::pre_skip_instructions() from reading the instr and
```

Comment:
Why this comment? Though true, it's slightly confusing because we're simply making the synthetic trace match what real traces have, which doesn't need justification. Or else, it looks like we're doing something different than real traces.

Reply:
It is indeed just making it match what real traces have.   Removed.